### DISCUSSION TITLE
Welcome to NanoVNA-Web-Client Discussions!

### DISCUSSION THREAD

**Discussion by cho45** [cho45/NanoVNA-Web-Client#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ref19665** [comment]
Hi 

I just got nanovna and I can't find a manual can anybody help

Also I have a blue. Tab sticking out .do I remove this? 
Thanks. In advance
